docker/Dockerfile

```diff
@@ -86,6 +86,13 @@ COPY --from=jre-build /javaruntime $JAVA_HOME
 COPY scripts /opt/bookkeeper/scripts
 RUN chmod +x -R /opt/bookkeeper/scripts/
 
+RUN chown -R 10000:0 /opt/bookkeeper
+RUN chmod -R ug+rwx /opt/bookkeeper
+RUN chmod -R g+rx /opt/bookkeeper/bin
+RUN chmod -R g+rx /opt/bookkeeper/scripts
```

Comment:
I tried to use the multi-staged, but the image size has not changed.

When we build the image, we need to use `--squash` feature, this can help us to reduce the intermediate image.

Reply:
> When we build the image, we need to use `--squash` feature, this can help us to reduce the intermediate image.

squashing is deprecated.

Reply:
> I tried to use the multi-staged, but the image size has not changed.

what did you try?

Reply:
> squashing is deprecated.

Good catch! I don't know this thing, because I'm using `podman` instead of `docker`, so I can still use the squash feature on the podman.

> what did you try?

I write two dockerfiles, one is multiple stages, and the other is a single stage, and then use the podman to build the different dockerfile:
```
cd $BK/docker
podman build --squash --platform linux/amd64 -t zixuan/bk:0822 .
```